src/state-manager/TransactionConsensus.ts

```diff
@@ -1220,6 +1238,31 @@ class TransactionConsenus {
     )
   }
 
+  verifyTwoThirdsMajority(receipt: AppliedReceipt2, executionGroupNodes: Set<string>): boolean {
+    const uniqueSignOwners = new Set(receipt.signatures.map(sign => sign.owner));
+    const totalNodes = executionGroupNodes.size;
+    const requiredMajority = Math.ceil((2 / 3) * totalNodes);
```

Comment:
Looks like we use a Math.round at consensus level: https://github.com/shardeum/shardus-core/blob/e8e32714bb86884f97aa387fcfd2b9e481cffb0e/src/state-manager/TransactionConsensus.ts#L1717-L1718